synctl

```diff
@@ -1326,6 +1420,47 @@ class SyntheticTest(Base):
         if self.ask_answer("are you sure to delete these tests?"):
             self.delete_multiple_synthetic_tests(delete_syn_id_lists)
 
+    def delete_a_credential(self, cred):
+        """Delete a credential"""
+        self.check_host_and_token(self.auth["host"], self.auth["token"])
+        if cred is None:
+            print("credential should not be empty")
+            return
+
+        credential = self.retrieve_credentials()
+
+        host = self.auth["host"]
+        token = self.auth["token"]
+
+        delete_url = f"{host}/api/synthetics/settings/credentials/{cred}"
+        headers = {
+            "Content-Type": "application/json",
+            "Authorization": f"apiToken {token}"
+        }
+
+        delete_res = requests.delete(delete_url, headers=headers,
+                                     timeout=60, verify=self.insecure)
+        if cred in credential:
```

Comment:
can you put the delete request under the if clause, only if cred in credentials list, synctl will send a delete request, otherwise exited.